### ISSUE TITLE
Medusa fails to start after most recent release

### ISSUE THREAD

**Issue by sammcj** [pymedusa/Medusa#5914]
- python 2.7.15
- centos 7
- global ffmpeg version is 3.4
- pip requirements are up to date

It looks like a problem with the x264 libs?

```
-- Unit medusa@apps.service has begun starting up.
Dec 25 13:59:32 nas python2.7[23350]: ffprobe: error while loading shared libraries: libx264.so.148: cannot open shared object file: No such file or directory
Dec 25 13:59:32 nas python2.7[23350]: Traceback (most recent call last):
Dec 25 13:59:32 nas python2.7[23350]: File "/opt/medusa/SickBeard.py", line 8, in <module>
Dec 25 13:59:32 nas python2.7[23350]: from medusa.__main__ import main
Dec 25 13:59:32 nas python2.7[23350]: File "/opt/medusa/medusa/__init__.py", line 24, in <module>
Dec 25 13:59:32 nas python2.7[23350]: initialize()
Dec 25 13:59:32 nas python2.7[23350]: File "/opt/medusa/medusa/init/__init__.py", line 35, in initialize
Dec 25 13:59:32 nas python2.7[23350]: _configure_knowit()
Dec 25 13:59:32 nas python2.7[23350]: File "/opt/medusa/medusa/init/__init__.py", line 218, in _configure_knowit
Dec 25 13:59:32 nas python2.7[23350]: api.initialize({'mediainfo': suggested_path})
Dec 25 13:59:32 nas python2.7[23350]: File "/opt/medusa/ext/knowit/api.py", line 27, in initialize
Dec 25 13:59:32 nas python2.7[23350]: available_providers[name] = provider_cls(config, context.get(name) or config.general.get(name))
Dec 25 13:59:32 nas python2.7[23350]: File "/opt/medusa/ext/knowit/providers/ffmpeg.py", line 202, in __init__
Dec 25 13:59:32 nas python2.7[23350]: self.executor = FFmpegExecutor.get_executor_instance(suggested_path)
Dec 25 13:59:32 nas python2.7[23350]: File "/opt/medusa/ext/knowit/providers/ffmpeg.py", line 88, in get_executor_instance
Dec 25 13:59:32 nas python2.7[23350]: executor = exec_cls.create(os_family, suggested_path)
Dec 25 13:59:32 nas python2.7[23350]: File "/opt/medusa/ext/knowit/providers/ffmpeg.py", line 117, in create
Dec 25 13:59:32 nas python2.7[23350]: check_output([candidate, '-version'])
Dec 25 13:59:32 nas python2.7[23350]: File "/usr/local/lib/python2.7/subprocess.py", line 223, in check_output
Dec 25 13:59:32 nas python2.7[23350]: raise CalledProcessError(retcode, cmd, output=output)
Dec 25 13:59:32 nas python2.7[23350]: subprocess.CalledProcessError: Command '[u'ffprobe', u'-version']' returned non-zero exit status 127
Dec 25 13:59:32 nas systemd[1]: medusa@apps.service: control process exited, code=exited status=1
Dec 25 13:59:32 nas systemd[1]: Failed to start Medusa Daemon.
```

**Comment by medariox** [comment]
Can you run ffmpeg from shell? Looks like there's something wrong with your ffmpeg install.

**Comment by sammcj** [comment]
Hmmm... it could be possible and if it is my own fault I do apologise in advance for logging what I thought was a bug, I have recompiled ffmpeg (successfully) on this system many times for various transcoding and ML projects.

Maybe the questions I should ask are:

- What does medusa use ffmpeg libraries for and what are Medusa’s requirements of ffmpeg?
- If there was an assumption made that my ffmpeg version is too new, Is there a way to link medusa to an older version of the required libraries?

```
 ~ ffmpeg
ffmpeg version 3.4 Copyright (c) 2000-2017 the FFmpeg developers
  built with gcc 4.8.5 (GCC) 20150623 (Red Hat 4.8.5-16)
  configuration: --prefix=/root/ffmpeg_build --pkg-config-flags=--static --extra-cflags=-I/root/ffmpeg_build/include --extra-ldflags=-L/root/ffmpeg_build/lib --extra-libs=-lpthread --bindir=/root/bin --enable-gpl --enable-libfdk_aac --enable-libfreetype --enable-libmp3lame --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree
  libavutil      55. 78.100 / 55. 78.100
  libavcodec     57.107.100 / 57.107.100
  libavformat    57. 83.100 / 57. 83.100
  libavdevice    57. 10.100 / 57. 10.100
  libavfilter     6.107.100 /  6.107.100
  libswscale      4.  8.100 /  4.  8.100
  libswresample   2.  9.100 /  2.  9.100
  libpostproc    54.  7.100 / 54.  7.100
Hyper fast Audio and Video encoder
usage: ffmpeg [options] [[infile options] -i infile]... {[outfile options] outfile}...
```

**Comment by medariox** [comment]
Medusa uses ffmpeg to determine the file quality if it is not otherwise possible (e.g. when the file name doesn't contain any quality information). I'd say the issue you are experiencing is a permission issue. Are you sure the user that runs Medusa also has the rights to access ffmpeg? The ffmpeg version shouldn't be the issue here.

**Comment by sammcj** [comment]
Thanks for your help and information.

While permissions were all fine, I just “fixed” my setup by adding `/usr/lib64/xpra` which appears to be missing from the default path for the application.

FYI - I added the following environment path to the systems unit:

```
Environment=LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib64/xpra/ 
```

So, I’m not certain if this was caused by me running a rebuilt ffmpeg - or by a medusa update, the weird thing is - I haven’t rebuilt ffmpeg in quite some time and it broke after the most recent merge from develop into master.

If you think it’s my ffmpeg builds fault - feel free to close this issue.
